src/axis.js

```diff
@@ -1,6 +1,5 @@
 import CLASS from './class';
-import { Component } from './core';
-import { isValue, isFunction, isString, isEmpty } from './util';
+import {isValue, isFunction, isString, isEmpty, Component } from './util';
```

Comment:
Component is exported by `./util` so you don't need to import it from `./core`  anymore